symex-interface-elisp.el

```diff
@@ -37,17 +38,6 @@
   "Eval entire containing definition."
   (eval-defun nil))
 
-(defun symex-eval-pretty-elisp ()
-  "Evaluate symex and render the result in a useful string form."
-  (interactive)
-  (symex-eval-elisp))
-
-(defun symex-eval-thunk-elisp ()
-  "Evaluate symex as a \"thunk,\" i.e. as a function taking no arguments."
-  (interactive)
-  ;; can use (eval (car (read-from-string thunk-code)))
-  (message "eval as thunk currently not supported for ELisp"))
-
 (defun symex-eval-print-elisp ()
   "Eval symex and print result in buffer."
   (interactive)
```

Comment:
nit: here as well, and above in `symex-eval-elisp`.

There seem to be a lot of these, so if you don't feel like including it in this PR, that's totally fine.